app/views/posts/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<%= render 'nav' %>
+
+<div class="d-flex m-0 p-0">
+    <div class="left col-12 p-3">
+        <%= render 'feed' %>    
+    </div>
+    <!-- <div class="right col-3 p-3 card my-3">
+        <%= render 'latest-post' %>
+    </div> -->
```

Comment:
It’s better to delete **commented piece of codes**. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))